go/teams/chatmessaging.go

```diff
@@ -96,6 +96,37 @@ func SendChatInviteWelcomeMessage(ctx context.Context, g *libkb.GlobalContext, t
 	return true
 }
 
+func SendChatSBSResolutionMessage(ctx context.Context, g *libkb.GlobalContext,
+	team, assertion string, prover keybase1.UID) (res bool) {
+
+	if !g.Env.SendSystemChatMessages() {
+		g.Log.CDebugf(ctx, "Skipping SentChatInviteWelcomeMessage via environment flag")
+		return false
+	}
+
+	proverName, err := g.GetUPAKLoader().LookupUsername(ctx, prover)
+	if err != nil {
+		g.Log.CDebugf(ctx, "SendChatSBSResolutionMessage: failed to lookup invitee username: %s", err)
+		return false
+	}
+
+	subBody := chat1.NewMessageSystemWithSbsresolve(chat1.
+		MessageSystemSbsResolve{
+		Prover:    proverName.String(),
+		Assertion: assertion,
```

Comment:
might want to parse the assertion here and pass serviceId / serviceUsername to the frontend